ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivityArgs.kt

```diff
@@ -1,29 +1,34 @@
 package com.revenuecat.purchases.ui.revenuecatui.activity
 
+import android.os.Build
 import android.os.Parcelable
 import com.revenuecat.purchases.ui.revenuecatui.fonts.ParcelizableFontProvider
 import com.revenuecat.purchases.ui.revenuecatui.fonts.PaywallFontFamily
 import com.revenuecat.purchases.ui.revenuecatui.fonts.TypographyType
 import kotlinx.parcelize.Parcelize
 
 internal const val DEFAULT_DISPLAY_DISMISS_BUTTON = true
+internal val defaultDisplayEdgeToEdge = Build.VERSION.SDK_INT >= Build.VERSION_CODES.VANILLA_ICE_CREAM
```

Comment:
I was actually thinking about this default... I changed it to also use edge to edge on android 35+, since that's the default starting on that version.